### DISCUSSION TITLE
Simulateur de stat

### DISCUSSION THREAD

**Discussion by Easygamer01s** [Crownicles/Crownicles#1914] [Nouvelles idées]
Bonjour à tous, 

Vu la nouvelle maj permettant de changer de classes, je me suis dis qu'il serait intéressant de remettre au gout du jour le simulateur de classes, qui n'est plus disponible sur le wiki il me semble... Ce dernier ne tenait pas compte des stats de l'équipement.. La prise en compte des stats pourrait se faire directement dans "/classinfo"..

Mais voici un petit simulateur qui devrait faire des heureux =D Voir son profil avec du Mythique, ça peut être sympa xD
[https://easygamer01s.shinyapps.io/SIMUL_DRAFT ](https://easygamer01s.shinyapps.io/SIMUL_DRAFT )

Toutes les classes sont présentes, tous les équipements apd exotique aussi .. enfin ceux dispo sur le wiki =P
Certains aspects comme le nerf des armes en fonction du niveau du joueur n'est pas encore pris en compte

Si cela vous plait, n'hésitez pas à mettre un lien dans le wiki/présentation du bot ? 

Cordialement,
Easy

**Comment by BastLast** [comment]
Oh le barjo

**Comment by BastLast** [comment]
N'importe qui peut modifier le wiki, donc tu peux rajouter le lien toi même !

**Reply by BastLast** [reply]
Par soucis de sécurité, avant de faire cela, est ce que le code est open source de ton application ?

**Reply by Easygamer01s** [reply]
C'est juste quelques heures sur R =D
Je pense que le code n'est pas directement disponible, mais je peux le partager..

**Comment by Easygamer01s** [comment]
La plateforme est dispo sur github en GNU General Public License v2.0
Donc c'est Open source si je comprend bien ^^
